### PR TITLE
DICT-459 Leeuwenhoek terms p2 

### PR DESCRIPTION
**Pull request by bullenc**
https://gdc-ctds.atlassian.net/browse/DICT-459 

Needs to be merged first before [some] clinical PRs can be submitted

### CODE REVIEW

File: src/gdcdictionary/schemas/_terms.yaml [hunk 1]
```diff
@@ -7581,3 +7591,105 @@ workflow_version:
...
+        Tumor of Origin Diagnosis Node
+      source:
+      cde_id:
+      cde_version:
+      term_url:
+
+prescribed_dose_units:
+  common:
+    description: >-
+      The units of the the amount of agent or medication formally prescribed to an
```
Comment (wwysoc2) [inline src/gdcdictionary/schemas/_terms.yaml]: Extra "the"

**Review by stilesj-uchicago — APPROVED** [review]
lgtm

**Comment by stilesj-uchicago** [comment]
Modify the title of the PR to lead with DAT-459 to get automatic association with the commits to the JIRA ticket.

**Review by wwysoc2 — APPROVED** [review]
Looks good, just fix small typo 👍

File: src/gdcdictionary/schemas/_terms.yaml (hunk 1 above)
Comment (bullenc) [reply src/gdcdictionary/schemas/_terms.yaml]: Fixed, thanks!

**Pull request merged**